joomla/joomla-cms
# [3.10] Handle the removal of the FTP support from Joomla 4 for com_joomlaupdate

### Summary of Changes

Handle the removal of the FTP support from Joomla 4 for com_joomlaupdate. At the time of the upload we can just add a message given that we dont know the version you are updating to at this point.

### Testing Instructions

- Install 3.10
- point to an Joomla 4 package
- extra: please make sure that on 3.x the option will be showed correctly

### Actual result BEFORE applying this Pull Request

You still have the FTP options

### Expected result AFTER applying this Pull Request

The FTP options are gone

### Documentation Changes Required

None
